eng/pipelines/notifications.yml

```diff
@@ -58,7 +58,7 @@ stages:
           notification-creator `
             --organization $(Organization) `
             --project $(Project) `
-            --path-prefix \"\\$(PathPrefix)\" `
+            --path-prefix "\$(PathPrefix)" `
```

Comment:
What's the motivation of making this change? It is working in my local. Do we have the change for simplicity?

Reply:
This escaping is not working in CI as it ends up with extra slashes and then pipelines fail to filter the pipelines. You can see some failures at https://dev.azure.com/azure-sdk/internal/_build/results?buildId=1240423&view=results.  I also had to fix the codeowner parsing with https://github.com/Azure/azure-sdk-tools/pull/2420.

Reply:
Thanks for clarification and I see what you mean! I have reviewed the changes for code owner parsing. 
<img width="451" alt="image" src="https://user-images.githubusercontent.com/48036328/145499216-a98b1120-8c8f-4570-9289-7878f61b10bd.png">